Controller/Payment/Callback.php

```diff
@@ -173,6 +176,8 @@ public function execute()
                 $payment = $order->getPayment();
                 $paymentId = $params['razorpay_payment_id'];
 
+                $rzpPayment = $this->rzp->request->request('GET', 'payments/'.$paymentId);
```

Comment:
@yashgit891 , why to poll payments API ? can't we use webhook to update this notes?

Reply:
yes, that works. In place of adding latency with callback.